1_turb_abl_fsi/template_files/NRELOffshrBsline5MW_Onshore_ElastoDyn_BDoutputs.dat

```diff
@@ -14,10 +14,17 @@ False         TeetDOF     - Rotor-teeter DOF (flag) [unused for 3 blades]
 False         DrTrDOF     - Drivetrain rotational-flexibility DOF (flag)
 False         GenDOF      - Generator DOF (flag)
 False         YawDOF      - Yaw DOF (flag)
+{if(TOWER)}

```

Comment:
also this won't work - because of OpenFAST - it expect certain inputs at very specific lines. It expects tower dofs at line 17.